openwisp-qa-format

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -e
 
+if which stylelint > /dev/null
+  then
+    stylelint $(find . -type f -name "*.css") --fix
+  else
+    echo "MODULE NOT FOUND: Please install stylelint"
+    exit 1
```

Comment:
once more time, I think this is wrong. Didn't we agree this feature must be backward compatible?

So why fail if `styleling` is not installed? This command must not fail if stylelint is missing. It could simply say "The program "stylelint" is not installed, Skipping CSS auto-formatting." or something similar.